wagtail/admin/views/generic/multiple_upload.py

```diff
@@ -171,6 +171,7 @@ def get_context_data(self, **kwargs):
             'help_text': self.form.fields['file'].help_text,
             'collections': collections,
             'form_media': self.form.media,
+            'collection_id': self.request.GET.get('collection_id', None),
```

Comment:
How would you feel about something more descriptive for this key, e.g. `selected_collection_id`? I think it might make the template conditionals a tad easier to read?

Reply:
@ababic no problem. I have followed naming convention in other templates, ie:

https://github.com/wagtail/wagtail/blob/dcd920d0e87c8f0399ebbfc14c4be89562df061f/wagtail/documents/templates/wagtaildocs/chooser/results.html#L58-L60

Reply:
@bmihelac This was only a minor nitpick to be fair. Happy to stick with `collection_id` if it's in-line with other things :) I'm notably a bit obsessive about descriptive naming.